src/state/slices/assetsSlice/assetsSlice.ts

```diff
@@ -90,8 +90,14 @@ export const assetApi = createApi({
         data && dispatch(assets.actions.setAssets(data))
       },
     }),
-    getAssetDescription: build.query<AssetsState, { assetId: AssetId; selectedLocale: string }>({
+    getAssetDescription: build.query<
+      AssetsState,
+      { assetId: AssetId | undefined; selectedLocale: string }
+    >({
       queryFn: async ({ assetId, selectedLocale }, { getState }) => {
+        if (!assetId) {
```

Comment:
Got burned by this one, in the end we don't end up even needing description for FOX farming but we might get burned by this in other consumers